setup.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
Ok. I think we should make sure to only have one set of instructions on the site at once, so please delete the old instructions when adding these. Ideally, put these in place of the old ones, so that the URL doesn't change, since we have links to it in various places.